### ISSUE TITLE
Borders contrasting for Internet Explorer users

### ISSUE THREAD

**Issue by imperator99** [govau/dta-gov-au#284]
# What’s the issue?

Users who browse in Internet Explorer and have a screen reader enabled will notice
focus issues – a purple border will appear directly outside a blue border when
navigating through to links (it is incorrectly rendering). For all other browsers, a small gap or a black border appears between the two. The contrast result it
provides when the purple and blue borders appear directly alongside each other
are:

|Outer border colour|Inner border colour|Current ratio|Required ratio|
|---|---|---|---|
|#D07AFF (Purple)|#43C2F6 (Blue)|1.3:1|3:1|

# Why it matters

Good contrast is useful for all users but is especially important for people with low vision and people viewing your website on a mobile device in a bright or sunny
location. The issue has been raised in this report as a functional issue due to it only occurring in only one browser. If it occurred across all browsers, the issue may potentially be identified as a fail for success criteria 1.4.11 Non-text Contrast under the proposed WCAG 2.1 guidelines.

# Our recommendation

Attempt to resolve the rendering issue which affects visible focus for Internet
Explorer users.
